openhtf/unittest_openhtf.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
I think we should consider creating a directory 'openhtf/test' and putting our unittest stuff in there.